### DISCUSSION TITLE
Msg de erro no chat id

### DISCUSSION THREAD

**Discussion by CleberSGoncalves** [elizandrodantas/bot-blaze-telegram#32] [General]
Amigo consegue me ajudar ja esta tudo funcionando, mas so consegue mandar msg para o bot e nao para o Canal que eu criei e adicionei o bot a ele (dei todas as permissoes de admin) ao bot dentro do canal. 

Mas quando o uso o codigo do bot (token): 6200675155:AAHMznssk5dNKdo8b82RBQEYqUFkLgCTdqo e o id do canal: -1796622231 ele apresenta a msg que "Chatid invalido ou bot nao tem permissao para enviar mensagem :("

![erroidchat](https://user-images.githubusercontent.com/31508893/220802643-47532e42-44a9-4f5e-ab66-cceebda37346.PNG)
Mas ja chequei tanto as permissoes do bot como as permissoes do bot dentro do canal. tem alguma Luz

**Comment by CleberSGoncalves** [comment]
Resolvido. tinha que usar o -100 antes do chatid. obrigado

**Reply by LuccaPraca** [reply]
amigo , estou com o mesmo problema , poderia explicar melhor como e esse -100?

**Reply by DinhoNunesLC** [reply]
Apenas digite -100 junto do id do canal.

'-100383737337'
